### PR TITLE
Endwalker

### PR DESCRIPTION
**Pull request by daemitus**
This should cover most of it. SMN is completely disabled.

### CODE REVIEW

**Comment by Yusman62** [comment]
Thank you for your hard work. Can't wait till the plugins are ready.

**Comment by erturpa** [comment]
@daemitus can you add swap BLM freeze/flare since it is now tied to their respective stance?

**Comment by Helios747** [comment]
Hey, small issue with this PR, MCH is slightly broken on it. When Hypercharge feature is enabled, it doesn't do anything.

Hypercharge doesn't get replaced with HB, but assigned as mutually exclusive buttons, still functions as stock.

**Comment by mkarer** [comment]
Awesome! Any idea though why it breaks the button/spell replacements that are done by the game itself? (Dosis -> Eukrasian Dosis when using Eukrasia as SGE, Gallows -> Cross Reaping while Shrouded as RPR)

**Comment by StevenT350** [comment]
For SMN: how about Energy Drain > Fester, Energy Siphon > Painflare and Summon Bahamut/Phoenix > Astral Flow ?

**Comment by hellozhewang** [comment]
For RDM's new acceleration, would it be possible to add an option to convert veraero 2 and verthunder 2 into scatter when acceleration is active? Similar how its doing it for dual cast here:
https://github.com/attickdoor/XIVComboPlugin/blob/master/XIVComboPlugin/IconReplacer.cs#L937

File: XIVComboPlugin/IconReplacer.cs [hunk 1]
```diff
@@ -928,7 +931,6 @@ private ulong GetIconDetour(byte self, uint actionID)
                 }
             }
 
-
             // Replace Redoublement with Redoublement combo, Enchanted if possible.
             if (Configuration.ComboPresets.HasFlag(CustomComboPreset.RedMageMeleeCombo))
                 if (actionID == RDM.Redoublement)
```
Comment (hellozhewang) [inline XIVComboPlugin/IconReplacer.cs:936]: Mana value for rdm combo changed.

**Comment by lovescandal** [comment]
Any possible ways that I can install the mod without using XIV Launcher?

**Comment by attickdoor** [comment]
Neat. I'll merge and build this shortly. I'll probably want to play RPR myself before implementing any more combos, and it'll take me a while to get familiar with what a bunch of other jobs do. Mostly I appreciate the sig updates and such.

**Comment by Zynyx** [comment]
Great work Attick! 

Any way you'd consider making the Summon Bahamut/Phoenix button swap to Enkindle Bahamut/Phoenix after its use?

**Pull request merged**